sdcm/cluster.py

```diff
@@ -1020,6 +1045,45 @@ def stop_scylla(self, verify_up=False, verify_down=True, timeout=300):
         self.stop_scylla_server(verify_up=verify_up, verify_down=verify_down, timeout=timeout)
         self.stop_scylla_jmx(verify_up=verify_up, verify_down=verify_down, timeout=timeout)
 
+    def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits=DEFAULT_MURMUR3_PARTITIONER_IGNORE_MSB_BITS):
```

Comment:
is `DEFAULT_MURMUR3_PARTITIONER_IGNORE_MSB_BITS` used elsewhere in the code or this is special for `restart_node_with_resharding`? If it is special for the function there is no need for a global variable definition.